torch/onnx/_internal/exporter/_compat.py

```diff
@@ -50,7 +50,7 @@ def export_compat(
     verbose: bool | None = None,
     input_names: Sequence[str] | None = None,
     output_names: Sequence[str] | None = None,
-    opset_version: int | None = None,
+    opset_version: int | None = _constants.TORCHLIB_OPSET,
```

Comment:
Should we delete the None and line 70 and 71 @justinchuby 
Is there other contexts?

Reply:
None is an allowed input in the main export api. So we accept it here